### DISCUSSION TITLE
[Manual] Updating Bare Metal install

### DISCUSSION THREAD

**Discussion by Burschi500** [jonaswinkler/paperless-ng#1030] [General]
I just did an update on a bare metal install, and in the first run i overwrote my paperless.conf. Maybe you could add some line to backup this file (i hope thats all to backup ;)), and to stop the paperless-* services.

**Comment by tandy-1000** [comment]
It would be nice to have a self updating mechanism, like LazyLibrarian and other selfhosted services.
